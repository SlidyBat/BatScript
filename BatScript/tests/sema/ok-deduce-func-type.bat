def int_func():
	return 1

def float_func():
	return 3.14

def str_func():
	return "Hello"

def bool_func():
	return false

def void_func():
	return

print int_func()
print float_func()
print str_func()
print bool_func()