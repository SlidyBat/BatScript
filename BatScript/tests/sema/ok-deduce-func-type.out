1
3.140000
Hello
false
